docs/source/release/v6.3.0/framework.rst

```diff
@@ -5,83 +5,73 @@ Framework Changes
+- Added the parameter ``MaxScatterPtAttempts`` to :ref:`DiscusMultipleScatteringCorrection <algm-DiscusMultipleScatteringCorrection>` to control number of attempts to generate initial scatter point.
 - :ref:`GenerateLogbook <algm-GenerateLogbook>` now allows to perform binary operations even when certain entries do not exist, e.g. to create a string with all polarisation orientations contained in a collection of data files.
 - Event nexuses produced at ILL can now be loaded using :ref:`LoadEventNexus <algm-LoadEventNexus>`.
-- :ref:`Rebin <algm-Rebin>` now has an option for binning with reverse logarithmic and inverse power bins.
-- :ref:`SetSampleFromLogs <algm-SetSampleFromLogs>` will now fail if the resulting sample shape has a volume of 0.
+- :ref:`Rebin <algm-Rebin>` now has an option ``UseReverseLogarithmic`` for binning with reverse logarithmic and inverse power bins.
+- :ref:`SaveAscii <algm-SaveAscii>` and :ref:`SaveCanSAS1D <algm-SaveCanSAS1D>` have a new property ``OneSpectrumPerFile``, controlling whether or not to save each spectrum in an individual file or all the spectra into a single file.
 - :ref:`SetSample <algm-SetSample>` can now load sample environment XML files from any directory using ``SetSample(ws, Environment={'Name': 'NameOfXMLFile', 'Path':'/path/to/file/'})``.
-- An importance sampling option has been added to :ref:`DiscusMultipleScatteringCorrection <algm-DiscusMultipleScatteringCorrection>` so that it handles spikes in the structure factor S(Q) better
-- Added parameter to :ref:`DiscusMultipleScatteringCorrection <algm-DiscusMultipleScatteringCorrection>` to control number of attempts to generate initial scatter point
-- Relative error option now enabled for peak table workspaces in :ref:`CompareWorkspaces <algm-CompareWorkspaces>`.
+- :ref:`SetSampleFromLogs <algm-SetSampleFromLogs>` will now fail if the resulting sample shape has a volume of 0.
 
 Bugfixes
 ########
-
-- Fix bug in :ref:`LoadEventNexus <algm-LoadEventNexus>` in checking valid event ID's and make sure to always exclude data in ``error`` and ``unmapped`` banks.
-- Fix bug in :ref:`Integration <algm-Integration>` when using UsePartialBinsOption with integration limits that are either equal or close together
-- Fix bug in :ref:`DiscusMultipleScatteringCorrection <algm-DiscusMultipleScatteringCorrection>` where calculation aborts with exception due to floating point rounding error when track segment close to vertical
-  Also fixed bug in calculation of track direction after scatter if pre-scatter track was pointing exactly down - sign of z component of new direction was incorrect
+- Fixed a bug with :ref:`CalculatePlaczek <algm-CalculatePlaczek>` algorithm for computing Placzek correction factors. There were a few flaws in the previously implemented formulation.
```

Comment:
Could we find out what those flaws were?
This sort of release note makes users nervous that their data might be wrong

Reply:
I looked back and this comes from PR #33173 . Looking at the description it seems the transformation constant from k to e was implemented wrong, as was the summation term formulation for second order Placzek corrections. Do we need more information that that? i.e. do we need to explain how they were wrong implemented?

Reply:
I would keep it at that, `previously implemented formula for transforming k to e, and the summation for second order Placezek corrections`

If people want more details they can reach out or look at the changes without us going into laborious (and possibly wrong) details